src/main/java/com/github/onsdigital/babbage/response/util/CacheControlHelper.java

```diff
@@ -56,7 +56,7 @@ private static void resolveHash(HttpServletRequest request, HttpServletResponse
                 .data("new_hash", newHash)
                 .log("resolving cache headers");
 
-        response.setHeader("Etag", newHash);
+        response.setHeader("Etag", "\"" + newHash + "\"");
```

Comment:
Just as an aside, it may have been clearer to use raw string literals to make this line cleaner. Not going to stop the approval though.
```suggestion
        response.setHeader("Etag", `"` + newHash + `"`);
```

Reply:
Actually ignore the above. I just noticed this was Java not Go 🤦‍♀️